spec/acceptance/candlepin_spec.rb

```diff
@@ -88,16 +103,29 @@
     describe command("keytool -list -keystore /etc/candlepin/certs/keystore -storepass $(cat #{keystore_password_file})") do
       its(:exit_status) { should eq 0 }
       its(:stdout) { should match(/^Keystore type: PKCS12$/i) }
-      its(:stdout) { should match(/^Your keystore contains 3 entries$/) }
+      its(:stdout) { should match(/^Your keystore contains 1 entries$/) }
       its(:stdout) { should match(/^tomcat, .+, PrivateKeyEntry, $/) }
-      its(:stdout) { should match(/^candlepin-ca, .+, trustedCertEntry, $/) }
     end
 
     describe command("keytool -list -v -keystore /etc/candlepin/certs/keystore -alias tomcat -storepass $(cat #{keystore_password_file})") do
       its(:exit_status) { should eq 0 }
       its(:stdout) { should match(/^Owner: CN=#{host_inventory['fqdn']}, OU=SomeOrgUnit, O=Katello, L=Raleigh, ST=North Carolina, C=US$/) }
       its(:stdout) { should match(/^Issuer: CN=#{host_inventory['fqdn']}, OU=SomeOrgUnit, O=Katello, L=Raleigh, ST=North Carolina, C=US$/) }
     end
+
+    describe command("keytool -list -keystore /etc/candlepin/certs/truststore -storepass $(cat #{truststore_password_file})") do
+      its(:exit_status) { should eq 0 }
+      its(:stdout) { should match(/^Keystore type: PKCS12$/i) }
+      its(:stdout) { should match(/^Your keystore contains 2 entries$/) }
+      its(:stdout) { should match(/^candlepin-ca, .+, trustedCertEntry, $/) }
+      its(:stdout) { should match(/^artemis-client, .+, PrivateKeyEntry, $/) }
```

Comment:
Looks like this is not a private key but a trusted cert
```suggestion
      its(:stdout) { should match(/^artemis-client, .+, trustedCertEntry, $/) }
```